frontend/cypress/e2e/FormStaffPageFuzzy.cy.ts

```diff
@@ -0,0 +1,839 @@
+          data.identificationProvinceValue,
+          false
+        );
+        cy.wait(10);
+      }
+
+      cy.fillFormEntry("#clientIdentification", data.clientIdentification);
+      cy.fillFormEntry("#birthdateYear", data.birthdateYear);
+      cy.fillFormEntry("#birthdateMonth", data.birthdateMonth);
+      cy.fillFormEntry("#birthdateDay", data.birthdateDay);
+    });
+
+    clickNext();
+  };
+
+  const fillRegistered = (extraData: any = {}) => {
+    cy.fixture("testdata/registeredBaseData").then((fixtureData: any) => {
+      const data = { ...fixtureData, ...extraData };
+
+      cy.get("#clientType").should("be.visible").and("have.value", "");
+      cy.selectFormEntry("#clientType", "BC registered business", false);
+
+      cy.selectAutocompleteEntry("#businessName", data.businessName, data.registrationNumber);
+
+      if(data.birthdateYear){
+        cy.fillFormEntry("#birthdateYear", data.birthdateYear);
+        cy.fillFormEntry("#birthdateMonth", data.birthdateMonth);
+        cy.fillFormEntry("#birthdateDay", data.birthdateDay);
+      }
+
+      if(data.doingBusinessAs){
+        cy.fillFormEntry("#doingBusinessAs", data.doingBusinessAs);
+      }
+
+      if(data.workSafeBcNumber){
+        cy.fillFormEntry("#workSafeBCNumber", data.workSafeBcNumber);
+      }
+
+      if(data.clientAcronym){
+        cy.fillFormEntry("#acronym", data.clientAcronym);
+      }
+    });
+
+    clickNext();
+  };
+
+  const fillFirstNations = (extraData: any = {}) => {
+    cy.fixture("testdata/firstNationsBaseData").then((fixtureData: any) => {
+      const data = { ...fixtureData, ...extraData };
+
+      cy.get("#clientType").should("be.visible").and("have.value", "");
+      cy.selectFormEntry("#clientType", "First Nation", false);
+
+      cy.selectAutocompleteEntry("#clientName", data.businessName, data.registrationNumber);
+    
+      if(data.workSafeBcNumber){
+        cy.fillFormEntry("#workSafeBcNumber", data.workSafeBcNumber);
+      }
+
+      if(data.clientAcronym){
+        cy.fillFormEntry("#clientAcronym", data.clientAcronym);
+      }
+    });
+
+    clickNext();
+  };
+
+  const fillOthers = (extraData: any = {}) => {
+    cy.fixture("testdata/otherBaseData").then((fixtureData: any) => {
+      const data = { ...fixtureData, ...extraData };
+
+      cy.get("#clientType").should("be.visible").and("have.value", "");
+      cy.selectFormEntry("#clientType", data.kind, false);
+
+      cy.fillFormEntry("#businessName", data.businessName);
+    
+      if(data.workSafeBcNumber){
+        cy.fillFormEntry("#workSafeBcNumber", data.workSafeBcNumber);
+      }
+
+      if(data.clientAcronym){
+        cy.fillFormEntry("#clientAcronym", data.clientAcronym);
+      }
+    });
+
+    clickNext();
+  };
+
+  const checkInputWarning = (element: string) => {
+    cy.get(element)
+    .should('be.visible')
+    .and('have.class', warningClass)
+    .shadow()
+    .find('input')
+    .should('not.have.class', errorClass);
```

Comment:
I think it's a little weird to be checking one class in the root element and the other in its child input.
If you did it because you didn't had the warning classname for the input, it's called "cds--text-input--warning".

Or maybe if we want to check only the root element, maybe we can check the error condition by asserting the presence of the attribute `invalid`.

But to be fair, it's working, and these helper functions are great! So please don't mind about this.

Reply:
I tried, but Cypress could not find it, even though I've copied the same code from the text input. It is weird but it works :man_shrugging:

About checking the invalid, my first version used to do that, but I decided to go with both CSS checks, even though in different levels